src/hooks/screen/useChatScreenHook.ts

```diff
@@ -177,7 +176,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
             currentChatSchema = await ChatSchema.generateSendingChat(
               randomId,
               userId,
-              selectedChannel?.id,
+              selectedChannel?.id || '',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The channel id is now provided with a fallback empty string value if `selectedChannel?.id` is undefined. This change could potentially lead to issues if the backend API does not handle empty string values appropriately for channel ids.
```diff
- selectedChannel?.id,
+ selectedChannel?.id || '',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->